CHANGELOG.md

```diff
@@ -8,6 +8,27 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.5.3] - 2024-04-16
+
+[Full changelog](https://github.com/openfga/openfga/compare/v1.5.2...v1.5.3)
+
+### Added
+
+* Apply tags to requests that have been intentionally throttled (https://github.com/openfga/openfga/pull/1531). This will add a new field "throttled" to such logs.
+
+### Fixed
+
+* Fix panic that occurred on Check API with some authorization models and tuples (https://github.com/openfga/openfga/pull/1517)
+
+### Changed
+
+* Remove experimentals for modular models/schema 1.2 (https://github.com/openfga/openfga/pull/1520)
+* Bumped to Go 1.21.9 (https://github.com/openfga/openfga/pull/1523)
+
+### Security
+
+* Patch CVE-2024-31452- a critical issue where Check API returns incorrect results for some models and tuples. See the CVE report for more details.
```

Comment:
Can you please make the CVE reference a markdown link to the published advisory?